tests/unit/text/test_keyword_extractor.py

```diff
@@ -29,38 +29,47 @@ def test_fit(self, corpus):
             extractor.fit()
 
     @pytest.mark.parametrize(
-        "n,exclude_words,expected",
+        "min_words,max_words,exclude_words,expected",
```

Comment:
This is not testing the knee detection, right?

Reply:
Not at its core, just whether the surrounding parameters work.

Reply:
I guess it makes more sense to see how it behaves in the notebooks anyway